modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/SnapshotPreconditionsValidator.scala

```diff
@@ -22,9 +22,9 @@ class SnapshotPreconditionsValidator[F[_]](config: SnapshotConfig) {
 
   def validateMinTipHeight(
     previousSnapshotHeight: Height,
-    tips: Tips
+    tips: Option[Tips]
   )(implicit N: Next[Height]): ValidationResult[Height] = {
-    val minTipHeight = tips.value.toList.minBy(_.height).height
+    val minTipHeight = tips.map(_.value.toList.minBy(_.height).height).getOrElse(Height(0))
```

Comment:
afaik there is `Height.Min`, so you can use it for `getOrElse`. If there is not, you can create one

Reply:
Fair point!